### PR TITLE
Find Pekka's clang

### PR DESCRIPTION
**Pull request by pvelesko**

### CODE REVIEW

**Comment by pvelesko** [comment]
@pjaaskel let me know if this fixes it for you

File: cmake/FindLLVM.cmake [hunk 1]
```diff
@@ -118,3 +118,4 @@ endif()
 enable_language(C CXX)
 # required by ROCm-Device-Libs, must be after project() call
 find_package(LLVM REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${CLANG_ROOT_PATH}/lib/cmake/llvm)
+find_package(Clang REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${CLANG_ROOT_PATH}/lib/cmake/clang)
```
Comment (pjaaskel) [inline cmake/FindLLVM.cmake:121]: Why NO_DEFAULT_PATH is there? It doesn't find it here if I have it set.

**Review by pjaaskel — CHANGES_REQUESTED** [review]

File: cmake/FindLLVM.cmake (hunk 1 above)
Comment (pvelesko) [reply cmake/FindLLVM.cmake:121]: We want to make sure we use the same Clang that's associated with the `llvm-config` that the user provided. many systems have a pre-installed Clang and we want to make sure we use matching versions.

This again would indicate to me that your Clang was built in some different way if LLVM cmake config and Clang cmake config reside in different directories.

Comment (pjaaskel) [reply cmake/FindLLVM.cmake:121]: OK. I'll investigate more.

**Pull request merged**